test/environment/README.md

```diff
@@ -0,0 +1,120 @@
+# Test Environment for azmi tool
+
+This PowerShell module is creating testing environment for azmi tool.
+https://github.com/SRE-PRG/azmitool
+
+## Contents
+
+- Basic Usage
+- Required permissions
+- Commands
+- Test VMs
+
+## Basic Usage
+
+Prior to using this PowerShell module you need to import it.
```

Comment:
Going step by step and will comments firs-go's perspective.
- Do I need admin rights? Usually sw requires it, so here could be good to tell user "no admin rights" required in a commentary.

Reply:
Updated